app/models/state_file_nj_intake.rb

```diff
@@ -202,7 +201,8 @@ def disqualifying_eligibility_rules
   def validate_state_specific_w2_requirements(w2); end
 
   def ask_for_signature_pin?
-    true
+    false
```

Comment:
[dust] this is false on the base intake, so you shouldn't need to define it

Reply:
Yep, we just thought it'd be clearer to define it again explicitly here.